test/state/state.cpp

```diff
@@ -307,6 +365,21 @@ std::variant<TransactionReceipt, std::error_code> transition(State& state, const
                    tx.to.has_value() ? tx.to.value() : bytes_view(), tx.value, tx.data,
                    tx.access_list, static_cast<bool>(tx.v), tx.r, tx.s);
     }
+    else
+    {
+        if (tx.v > 1)
+            throw std::invalid_argument("`v` value for blob transaction must be 0 or 1");
+        if (!tx.to.has_value())  // Blob tx has to have `to` address
+            throw std::invalid_argument("`to` value for blob transaction must not be null");
```

Comment:
Codecov says this is uncovered by tests. Please add a test if true.

Reply:
Test addded